dvc/repo/destroy.py

```diff
@@ -1,3 +1,4 @@
+from dvc.ignore import destroy as destroy_dvcignore
```

Comment:
Nitpick: You can import it as:
```suggestion
from dvc import ignore

ignore.destroy()
```